### PR TITLE
Get rid of bash

### PR DESCRIPTION
**Pull request by RomanHotsiy**

### CODE REVIEW

File: deploy-to-gh-pages.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+if (!process.env.GH_TOKEN) {
+  console.error('You need to setup GH_TOKEN environment variable')
+  process.exit(1);
+}
+
+var folder = process.argv[2];
+var GH_REPO=execAndLog('git config --get remote.origin.url').stdout;
+
+if (GH_REPO.substring(0, 8) !== 'https://') {
```
Comment (IvanGoncharov) [inline deploy-to-gh-pages.js]: startWith?

File: deploy-to-gh-pages.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+console.log('+ cd ' + folder);
+cd(folder);
+execAndLog('git init')
+execAndLog('git config user.name "Travis-CI"');
+execAndLog('git config user.email "travis@travis"');
+
+execAndLog('git add .');
+execAndLog('git commit -m "Deployed to Github Pages"');
+
+var GH_URL= GH_REPO.replace('://', '://' + process.env.GH_TOKEN + '@').trim();
```
Comment (IvanGoncharov) [inline deploy-to-gh-pages.js]: why you need trim?

File: deploy-to-gh-pages.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+try {
+  execAndLog('git push --force "' + GH_URL + '" master:gh-pages');
+} catch(e) {
```
Comment (IvanGoncharov) [inline deploy-to-gh-pages.js]: make try catch global

File: deploy-to-gh-pages.js (hunk 1 above)
Comment (RomanHotsiy) [inline deploy-to-gh-pages.js]: not compatible with node 0.12

File: deploy-to-gh-pages.js (hunk 2 above)
Comment (RomanHotsiy) [inline deploy-to-gh-pages.js]: because `GH_REPO` goes from stdout with ending newline

File: deploy-to-gh-pages.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  execAndLog('git add .');
+  execAndLog('git commit -m "Deployed to Github Pages"');
+
+  var GH_URL= GH_REPO.replace('://', '://' + process.env.GH_TOKEN + '@').trim();
+  execAndLog('git push --force "' + GH_URL + '" master:gh-pages');
+}
+
+try {
+  doRelease();
+} catch(e) {
```
Comment (IvanGoncharov) [inline deploy-to-gh-pages.js:50]: :+1:

**Pull request merged**